designs/tripleo/osc-tripleo.md

```diff
@@ -52,25 +52,25 @@ description: >
   Open Security Controller: service for orchestration of virtual network security functions.
 
  parameters:
-  #  The paramater EndpointMap is required for all service templates*. TODO: Clarify how/why this is needed.
+  #  The paramater EndpointMap is required for all service templates*. It provides the location of the service endpoint, i.e,: IP address and port.  
   EndpointMap:
     default: {}
     description: Mapping of service endpoint -> protocol. Typically set
                  via parameter_defaults in the resource registry.
 				 
   # The name of the OSC image in the Kolla repository**.
   OSCDockerImage:
-    description: image
+    description: The name of the OSC container image to be deployed.
     type: string
 
   # Placeholder to exemplify an OSC environment variable.
   OSCEnvVar:
-    description: image
+    description: Placeholder/sample parameter for an environment variable used by the OSC container.
     type: string
 	
-  # The .
+  # The source of the mounted volume OSC needs to persist information.
   OSCVolumeSrc:
-    description: image
+    description: The source of the mounted volume OSC needs to persist information.
```

Comment:
is there a volume destination? Where does docker expose the src volume? Docker run for example takes a target

`docker run -p 443:443 -p 8090:8090 -dit --mount source=osc-vol,target=/opt/vmidc/bin/data`

Reply:
Yes, there is a target but currently not parameterized though `{get_param: OSCVolumeSrc}:/opt/vmidc/bin/data/`